avocado_vt/plugins/vt_joblock.py

```diff
@@ -123,8 +106,9 @@ def pre(self, job):
                 self._lock(job)
         except Exception as detail:
             msg = "Failure trying to set Avocado-VT job lock: %s" % detail
-            self._abort(msg, job)
+            self.log.error(msg)
+            sys.exit(exit_codes.AVOCADO_JOB_FAIL | job.exitcode)
 
     def post(self, job):
         if self.lock_file is not None:
-            self._unlock()
+            os.unlink(self.lock_file)
```

Comment:
Maybe this is a problem with the job pre/post execution of `post()`? In theory, plugin writers shouldn't need to think about that, so maybe the real fix should go somewhere else.